spec/requests/afford_food.rb

```diff
@@ -0,0 +1,52 @@
+RSpec.describe "afford-food" do
+  describe "GET /afford-food" do
+    let(:selected_option) { I18n.t("coronavirus_form.questions.afford_food.options").sample }
+
+    context "without session data" do
+      it "shows the form" do
+        visit are_you_off_work_ill_path
```

Comment:
```suggestion
        visit afford_food_path
```

Reply:
Oops! Copy and Paste error 🤦‍♂

Reply:
It's a bit worrying that the test passed. 🤔

Reply:
Because I'd named the file incorrectly 🤦‍♂